extensions/amp-story/amp-story.md

```diff
@@ -785,6 +785,62 @@ You can apply multiple entrance animations on one element (for example, an eleme
 If a composed animation is supposed to start after the end of a separate element's animation, make sure that all nested elements that compose the animation have the attribute `animate-in-after` set to the same `id`.
 {% endcall %}
 
+## Branching
+
+Stories support fragment paramenters in the url. By utilizing this, you can create stories that start from somewhere other than the beginning, share specific pages of stories, or even allow users to jump around within a story. An example is a table of contents.
+
+### Manipulating Navigation
+
+The feature allows you to manipulate navigation within a story. When using it, navigation throughout the story from a 'branch-off point' will follow the path that a user takes. If a user navigates from `page-1` to `page-5` and then `page-6`, navigating backwards will follow this exact path (`page-6` to `page-5` and finally `page-1`). The other pages will not be exposed to the user.
+
+### Sharing Options
+
+Specific pages within a story are shareable through URLs.
```

Comment:
Some uses of `url` are lowercase and other are capitalized. Can we please capitalize all of them?

Reply:
I'd rephrase to:
User's can share a specific page within a story through URLs.

Reply:
Will we be providing a share button for this feature?

Reply:
I don't think so. I just understood what Gabriel was saying, and that we are not actually going to be generating the URLs as the user navigate through the story. 

But the publisher (or even a user if they know the page ID) could share the story starting on a specific page by appending the fragment parameter in the URL